scss/components/_grid.scss

```diff
@@ -15,3 +15,11 @@
 .container-fluid {
   max-width: $layout-max-width;
 }
+
+.row {
+  margin-top: $grid-gutter-width * -1;
```

Comment:
Wouldn't **-$grid-gutter-width** have the same effect?

Reply:
Yup :)